pkg/rpc/response/result/application_log.go

```diff
@@ -25,6 +27,73 @@ type NotificationEvent struct {
 	Item     smartcontract.Parameter `json:"state"`
 }
 
+type applicationLogAux struct {
+	TxHash      util.Uint256        `json:"txid"`
+	Trigger     string              `json:"trigger"`
+	VMState     string              `json:"vmstate"`
+	GasConsumed int64               `json:"gasconsumed,string"`
+	Stack       json.RawMessage     `json:"stack"`
```

Comment:
Is there anything preventing making it a `[]json.RawMessage` and saving on manual `[]json.RawMessage -> json.RawMessage` serialization?

Reply:
Nothing, fixed.